# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1313672603 -7200
#      Thu Aug 18 15:03:23 2011 +0200
# Node ID e0a8e35e750e56d019f5c8eeb737655b1bc5d3c4
# Parent  f03f4ac63569d76987e2c83691460137c9751b87
Now scrolling down in terminal when typing.

diff --git a/gui/src/terminal/TerminalDisplay.cpp b/gui/src/terminal/TerminalDisplay.cpp
--- a/gui/src/terminal/TerminalDisplay.cpp
+++ b/gui/src/terminal/TerminalDisplay.cpp
@@ -2607,16 +2607,17 @@ TerminalDisplay::keyPressEvent (QKeyEven
       else
 	_cursorBlinking = false;
     }
 
   if (emitKeyPressSignal)
     emit keyPressedSignal (event);
 
   event->accept ();
+  _scrollBar->setValue (_scrollBar->maximum ());
 }
 
 void
 TerminalDisplay::inputMethodEvent (QInputMethodEvent * event)
 {
   QKeyEvent keyEvent (QEvent::KeyPress, 0,
 		      (Qt::KeyboardModifiers) Qt::NoModifier,
 		      event->commitString ());
